crest/Assets/Crest/Crest-Examples/BoatDev/Data/WavesWakeRig.asset

```diff
@@ -0,0 +1,69 @@
+%YAML 1.1
```

Comment:
Missed this one. Probably should be renamed to WavesBoatWakes.

Reply:
ah thanks done